src/assets/tokens/tokens.styl

```diff
@@ -203,3 +203,18 @@ $duration-normal = 1s
 $duration-long = 2s
 /* Longest time duration, very pronounced */
 $duration-longest = 4s
+/* Lowest stacking level */
+$z-index-bottom = 1
```

Comment:
The lowest stacking level, if I'm not mistaken, is the default of zero (0), is it not?